dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/feature/FeatureScope.kt

```diff
@@ -9,12 +9,18 @@ package com.datadog.android.api.feature
 import androidx.annotation.AnyThread
 import com.datadog.android.api.context.DatadogContext
 import com.datadog.android.api.storage.EventBatchWriter
+import com.datadog.android.api.storage.datastore.DataStoreHandler
 
 /**
  * Represents a Datadog feature.
  */
 interface FeatureScope {
 
+    /**
+     * Property to enable interaction with the DataStore.
+     */
+    var dataStore: DataStoreHandler
```

Comment:
blocking: It should be immutable property here, we don't want to have a possibility to modify it at the call site.

```suggestion
    val dataStore: DataStoreHandler
```